app/src/lib/click.ts

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect } from "react"
```

Comment:
would rename this file `useOnClickOutside` like useInterval, and maybe leave a comment w/this reference code - https://usehooks.com/useOnClickOutside/

and newline at EOF

Reply:
Done